conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/GramineEnclaveEnvironment.kt

```diff
@@ -9,10 +9,11 @@ import java.security.MessageDigest
 /**
  * This class is the enclave environment intended for use with gramine-direct.
  */
-class GramineDirectEnclaveEnvironment(
-        enclaveClass: Class<*>,
-        override val hostInterface: SocketEnclaveHostInterface,
-        private val mrsigner: ByteArray
+class GramineEnclaveEnvironment(
+    enclaveClass: Class<*>,
+    override val hostInterface: SocketEnclaveHostInterface,
+    private val simulationMrSigner: ByteArray,
```

Comment:
Can you use `SecureHash` to make this a bit more typesafe. It has a `buffer()` method which you can use when setting the `mrsigner` field.